Count/ViewController.swift

```diff
@@ -10,47 +10,65 @@ import UIKit
 
 class ViewController: UIViewController {
     
-    var number: Int = 0
     @IBOutlet var label: UILabel!
-
+    @IBOutlet var bestView: BestLabel!
+    private var timer: Timer?
+    private var timerTotalDuration: TimeInterval = 0
+    var db = UserDefaults.standard
```

Comment:
```suggestion
    let dataBase = UserDefaults.standard
```

もしくは

```suggestion
    let saveData = UserDefaults.standard
```

かなぁ...UserDefaultsはデータベースというよりはキャッシュだから教科書にあわせたほうがよいかも
あとここは変えないからletかな